### DISCUSSION TITLE
printout document.getText() from debug view on a Mac

### DISCUSSION THREAD

**Discussion by ArturoDent** [microsoft/vscode-discussions#1399] [Extension Development QnA] [answered]
Could someone - on a Mac - please print out the document.getText() debug value of a few lines (including one empty line) please.   Any language is fine.  I know vscode does some normalizing of line endings and I need to see what the actual debugging text value it reports.  In the debug view make an entry for docuemnt.getText(with a range of ~4 lines including one empty line) and copy the `value` and paste it here - it should include literal `\n`'s, etc.  Thank you.

**Comment by robconery** [comment, marked as answer]
Hi Arturo - if you have a specific question we can try to help. Otherwise, the line endings _should_ be independent of the operating system. Either way: a specific query helps.
